### DISCUSSION TITLE
Force restore on same namespaces

### DISCUSSION THREAD

**Discussion by cloudcafetech** [vmware-tanzu/velero#3921] [Community support Q&A]
Is it possible to do force restore on same namespaces in all objects?

**Comment by zubron** [comment]
Velero will only attempt to restore items that are not present in the cluster and there is currently no way to force the restore. There is an open issue for this (#469) and it is in our [roadmap](https://github.com/vmware-tanzu/velero/blob/main/ROADMAP.md) but this work is not currently prioritized.
